bots/tests-policy

```diff
@@ -221,6 +221,9 @@ def normalize_traceback(trace):
     # replace noise in SELinux violations
     trace = re.sub('audit\([0-9.:]+\)', 'audit(0)', trace)
     trace = re.sub(r'\b(pid|ino)=[0-9]+ ', r'\1=0 ', trace)
+
+    # in Python 3, testlib.Error is shown with namespace
+    trace = re.sub('testlib\.Error', 'Error', trace)
```

Comment:
I'd strongly consider normalising this in the other direction (and updating the naughties).  It seems that py3 is the expected norm now, so why not treat it that way?

Reply:
I filed issue #9591 about it. I'm still hoping that the actually printed format can be fixed to just `Error:` again, I just didn't see a good way of doing this. I don't want to change all the naughties until this settled down and we also port the selenium, avocado, and container tests.